src/main/java/seedu/address/model/candidate/UniqueCandidateList.java

```diff
@@ -135,4 +135,15 @@ private boolean candidatesAreUnique(List<Candidate> candidates) {
         }
         return true;
     }
+
+    /**
+     * Resets the interviewStatus of all candidates whose interview statuses are scheduled to not scheduled.
+     */
+    public void resetScheduledStatus() {
+        for (Candidate c : internalList) {
```

Comment:
Hello @lzan98 for `schedule clear` will we also need to check whether the interview date has passed? because right now, the only time that `interviewstatus` is triggered to `complete` is when expired interview dates are manually deleted/automatically deleted at starting up. i.e. if we clear the interview, we might be deleting expired interviews as well, hence might need to check for this and change the `interviewstatus` to `complete` where relevant?